### PR TITLE
add Gateway mode extension

### PR DESCRIPTION
**Pull request by wxbty**
## What is the purpose of the change

By using the ByteAccessor extension, the dubbo application can receive requests when service not found exceptions occur, and customize the processing logic. A gateway mode for dubbo applications
relate  #https://github.com/apache/dubbo/pull/11088

### CODE REVIEW

**Comment by wxbty** [comment]
@AlbumenJ  PTAL

File: dubbo-gateway-extensions/dubbo-gateway-consumer/src/main/java/org/apache/dubbo/gateway/consumer/filter/OmnSerFilter.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+    public static final String name = "specifyAddress";
+
+    @Override
+    public Result invoke(Invoker<?> invoker, Invocation invocation) throws RpcException {
+
+        Object address = invocation.get(name);
+        if (address != null) {
+            convertParameterTypeToJavaBeanDescriptor(invocation);
+        }
```
Comment (AlbumenJ) [inline dubbo-gateway-extensions/dubbo-gateway-consumer/src/main/java/org/apache/dubbo/gateway/consumer/filter/OmnSerFilter.java:51]: Conside to add a flag in RpcContext.clientAttachment to indicate that the call is omn call

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

File: dubbo-gateway-extensions/dubbo-gateway-consumer/src/main/java/org/apache/dubbo/gateway/consumer/filter/OmnSerFilter.java (hunk 1 above)
Comment (wxbty) [reply dubbo-gateway-extensions/dubbo-gateway-consumer/src/main/java/org/apache/dubbo/gateway/consumer/filter/OmnSerFilter.java]: ok

**Comment by AlbumenJ** [comment]
image

**Comment by wxbty** [comment]
> image

done

**Pull request merged**